client/rest/resources/rest.light.json

```diff
@@ -0,0 +1,62 @@
+{
+    "network": {
+        "name": "testnet",
+        "description": "catapult public test network"
+    },
+
+    "port": 3000,
+    "protocol": "HTTPS",
+    "sslKeyPath": "",
+    "sslCertificatePath": "",
+    "crossDomain": {
+        "allowedHosts": ["*"],
+        "allowedMethods": ["GET", "POST", "PUT", "OPTIONS"]
+    },
+
+    "routeExtensions": [
+        "light"
```

Comment:
Should light be under extensions vs routeExtensions? 🤔

Reply:
hmm, I think it make sense.
as `light` is a REST route, instead of a new plugins

Reply:
Ok, should this route always be loaded instead of an extension?

Reply:
> Ok, should this route always be loaded instead of an extension?
yes, we no need anything extension.